src/Core/MailTemplates/Handlebars/SecretsManagerAccessRequest.html.hbs

```diff
@@ -0,0 +1,27 @@
+{{#>FullHtmlLayout}}
+
+<table width="100%" cellpadding="0" cellspacing="0" style="margin: 0; font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif; box-sizing: border-box; font-size: 16px; color: #333; line-height: 25px; -webkit-font-smoothing: antialiased; -webkit-text-size-adjust: none;">
+    <tr style="margin: 0; font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif; box-sizing: border-box; font-size: 16px; color: #333; line-height: 25px; -webkit-font-smoothing: antialiased; -webkit-text-size-adjust: none;">
+        <td class="content-block" style="font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif; box-sizing: border-box; font-size: 16px; color: #333; line-height: 25px; margin: 0; -webkit-font-smoothing: antialiased; padding: 0; -webkit-text-size-adjust: none;" valign="top">
+            {{{UserNameRequestingAccess}}} has requested access to secrets manager: <br /><br />
```

Comment:
`{{{UserNameRequestingAccess}}}`

Triple stash `{{{` tells handlebars not to HTML-escape the input. 

https://handlebarsjs.com/guide/expressions.html#html-escaping

Typically, we use `{{` for everything and only use triple for generated URLs so everything is properly HTML escaped.